build/setup-kubebuilder.sh

```diff
@@ -2,6 +2,8 @@
 
 set -e
 
+export KUBEBUILDER_VERSION=4.8.0
```

Comment:
How about replacing the entire script with

```
go install sigs.k8s.io/kubebuilder/v4@latest
```

Reply:
Ah, indeed. We wouldn't want to get bitten unexpectedly by breaking major version updates. I'll make those module paths static. Though perhaps it could detect a major version update and create an opt-in flow during the transition. 🤔